components/eamxx/cime_config/testdefs/testmods_dirs/scream/internal_diagnostics_level/shell_commands

```diff
@@ -0,0 +1,9 @@
+# Can't change atm_proc_base::internal_diagnostics_level, so change each one
```

Comment:
I think this has been fixed in #2334 , so you should be able to just do
```
./xmlchange --append SCREAM_ATMCHANGE_BUFFER=" --all internal_diagnostics_level=1"
```
You should probably try it first, though.

Reply:
Looks like it's not working for my use case. I need internal_diagnostics_level for the EAMxx-labeled process to be 0, but I can't seem to specify anything like that. For example, this doesn't seem to work:
```
./atmchange --all internal_diagnostics_level=1 atmosphere_processes::internal_diagnostics_level=1
```
And `EAMxx::internal_diagnostics_level` doesn't exist; I seem to recall it did previously. My original shell_commands still works, though, so I'm going to stick with that.